### PR TITLE
Driving topic

### LINKED ISSUE #58: [ FEATURE REQUEST ] - Natsuki and driving
# Overview:
This feature will introduce a new topic prompted by the player, allowing them to ask Natsuki about whether she can drive.

## Must haves:

- New topic "Driving"; where Natsuki will talk to the player about her (inability!) to drive.

## Could haves:
N/A

## Is this feature request related to a problem? Please describe.
This is a new feature.

### PR DESCRIPTION
**Pull request by Blizzardsev**
This will introduce the following:

- One new topic, `talk_driving`.

Closes #58

### CODE REVIEW

**Review by spaghetsie — APPROVED** [review]
patpatpatpaptpatpat

File: game/script-topics.rpy [hunk 1]
```diff
@@ -1468,6 +1468,75 @@ label talk_drinking_alcohol:
...
+    n "Ahaha!{w=0.2} What kind of a question is that,{w=0.1} [player]?"
+    $ chosen_tease = random.choice(jn_globals.DEFAULT_PLAYER_TEASE_NAMES)
+    n "Of course I can't drive,{w=0.1} [chosen_tease]!{w=0.2} Why do you think I walk everywhere?"
+    n "I mean...{w=0.3} even if I wanted to learn,{w=0.1} I don't think I could afford it."
+    n "Lessons are super expensive nowadays!"
+    n "And then there's tests,{w=0.1} insurance...{w=0.3} it's actually pretty gross how fast it all adds up."
+    n "I think I'd rather stick to public transport and my own two feet."
+    n "But what about you,{w=0.1} [player]?"
+    menu:
+        "Can you drive?"
```
Comment (JustNatsFang) [inline game/script-topics.rpy:1499]: Need to make sure Nat says this
```suggestion
        n "Can you drive?"
```

**Review by JustNatsFang — CHANGES_REQUESTED** [review]
something

**Review by JustNatsFang — APPROVED** [review]
patt

**Pull request merged**